File: Data_Structure/python/17219.py
import sys
input = sys.stdin.readline
n,m = map(int,input().rstrip().split())
dic = {}
for _ in range(n):
  id_,password_ = input().split()
  dic[id_] = password_
result = ""
for _ in range(m):
  id_ = input().rstrip()
  result += dic[id_]+ "\n"
print(result)

File: Data_Structure/javascript/17219.js
const input = require("fs").readFileSync("/dev/stdin").toString().split("\n");

const [n, m] = input[0].split(" ").map(Number);
const dic = new Map();
let id_, password_;
let i = 1;
for (; i <= n; i++) {
  [id_, password_] = input[i].split(" ");
  dic.set(id_, password_);
}
let result = "";
for (; i < input.length - 1; i++) {
  id_ = input[i];
  result += dic.get(id_) + "\n";
}
console.log(result);
